control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -3305,7 +3308,6 @@ components:
         - uuid
         - size
         - source_volume
-        - timestamp
```

Comment:
Do we even need the timestamp for VolumeSnapshotState? Csi will consume from metadata. And in state we have the individual replica snapshot state time stamp.

Reply:
hmm we might not even need it yes